pom.xml

```diff
@@ -134,6 +134,11 @@
           <version>0.9.1</version>
           <scope>test</scope>
       </dependency>
+      <dependency>
+          <groupId>com.mysql</groupId>
+          <artifactId>mysql-connector-j</artifactId>
+          <version>8.1.0</version>
+      </dependency>
```

Comment:
I think we should remove postgresql driver dependencies from pom.xml, because we are not using it.